src/spyglass/position/v1/dlc_utils_makevid.py

```diff
@@ -31,6 +31,8 @@
     "#ffe91a",
 ]
 
+matplotlib.use("Agg")  # Use non-interactive backend
```

Comment:
Do you know if this will affect the context after importing? Will the user have to reset this to see plots?

Reply:
Good point - My latest commit tracks the previous backend and reverts on completion